docs/source/getting-started.mdx

```diff
@@ -103,7 +109,7 @@ The above `start` script tells TypeScript to compile your code into JavaScript b
 
 ```
 > graphql-server-example@1.0.0 start
-> tsc && node ./dist/index.js
+> npm run compile && node ./dist/index.js
```

Comment:
@rkoron007 this is now equivalent to `npm start` (based on the start script you added above)